src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -92,9 +92,14 @@ def test_WINNF_FT_S_SIQ_3(self):
 
     # Inject GWBL station operating at range 3650 - 3700 MHz within 150 km of
     # FSS (approx. 100 km from FSS location set above).
-    gwbl = json.load(
+    gwbl_data = json.load(
         open(os.path.join('testcases', 'testdata', 'gwbl_record_0.json')))
-    self._sas_admin.InjectWisp(gwbl)
+    self._sas_admin.InjectWisp(gwbl_data)
+    gwbl = gwbl_data['record']
+    gwbl_low_frequency = gwbl['deploymentParam'][0]['operationParam'][
```

Comment:
The GWBL frequency is irrelevant since we protect the top 50 MHz (3650-3700) regardless of the GWBL frequency. Please revert this and the related changes.

Reply:
From the SAS Admins agreement: "If the CBSD is within 150 km of an FSS station operating below 3700MHz, and a GWBL station is also within 150 km of the FSS station (using any portion of the channel), SAS shall declare the 3650-3700 MHz as unavailable. (This applies to the Grant procedure as well.)"